docs/source/examples/ucx.rst

```diff
@@ -69,6 +72,9 @@ To start a cluster with all supported transports and an RMM pool:
     > --net-devices="auto" \
     > --rmm-pool-size="1GB"
 
+.. note::
+    For UCX 1.9 (deprecated) and older it's also necessary to set ``--net-devices="auto"``. UCX 1.11 and above is capable of selecting InfiniBand devices automatically.
```

Comment:
Think for sake of consistency, we'll also want to remove `--net-devices="auto"` from the above code block

Reply:
Good catch, done in https://github.com/rapidsai/dask-cuda/pull/720/commits/e2b3e04cdf39f9405e221dbdb409a2dff41511bc .